services/beaconchain/src/main/java/tech/pegasys/artemis/services/beaconchain/BeaconChainController.java

```diff
@@ -358,7 +354,7 @@ private void processSlot() {
         EVENT_LOG.epochEvent();
       }
 
-      this.eventBus.post(new SlotEvent(nodeSlot));
+      slotEventsChannelPublisher.onSlot(new SlotEvent(nodeSlot));
```

Comment:
nit: We probably don't need `SlotEvent` anymore - can just call `onSlot(nodeSlot)`.  The dedicated event was just because `EventBus` didn't provide any other context so you had to search for all usages of whatever you posted, whereas the event channel interface makes it clear where this is being used.

Reply:
Makes sense; will take that out now.

Reply:
Done.